eng/release/DiagnosticsReleaseTool/DiagnosticsReleaseTool.csproj

```diff
@@ -22,7 +22,7 @@
 
     <PackageReference Include="Azure.Identity" Version="[1.12.0]" />
     <PackageReference Include="Azure.Storage.Blobs" Version="[12.21.2]" />
-    <PackageReference Include="System.CommandLine" Version="2.0.0-beta1.20468.1" />
+    <PackageReference Include="System.CommandLine" Version="2.0.0-beta1.20478.1" />
```

Comment:
Why the version change?

Reply:
I think the older version was in one of the older dotnetX feeds that I removed so it could find it anymore. I bumped it to the next available version. We should probably update this project to use CPM, but want to leave that for another change.